### PR TITLE
upgrade-and-coexistence-of-skypeforbusiness-and-teams.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue #1762 - Situations mentioned under the section "Teams Only" mode are not relevant to it. So removed the two points.

### CODE REVIEW

**Comment by LolaJacobsen** [comment]
@bjwhalen and @dearbeen - please review these proposed changes and let me know whether to merge the PR.

**Comment by scanum** [comment]
nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/upgrade-and-coexistence-of-skypeforbusiness-and-teams.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following table compares coexistence and upgrade modes.
...
+|Teams Only |You might have already deployed users in islands mode and are ready to retire Skype for Business for groups of users |Reduce variable costs on Skype for Business (on-premises server operations, outsourcing contract, and so on)<br><br>Go to the full Teams experience as quickly as possible, for at least some users|Limits user confusion by providing only one client to work with Interoperability with users in Skype for Business Only, Skype for Business with Teams Collaboration, Skype for Business with Teams Collaboration and Meetings|Interoperability only supports basic chat and calling between Skype for Business and Teams, and interop escalation scenarios for desktop sharing and multi-party chat and calling|
```
Comment (JohanFreelancer9) [inline Teams/upgrade-and-coexistence-of-skypeforbusiness-and-teams.md]:
```suggestion
|Teams Only |You might have already deployed users in Islands mode and are ready to retire Skype for Business for groups of users |Reduce variable costs on Skype for Business (on-premises server operations, outsourcing contract, etc.)<br><br>Go to the full Teams experience as quickly as possible, for at least some users |Limits user confusion by providing only one client to work with Interoperability with users in Skype for Business only, Skype for Business with Teams collaboration, Skype for Business with Teams collaboration and Meetings |Interoperability only supports basic chat and calling between Skype for Business and Teams, and Interoperability escalation scenarios for desktop sharing and multi-party chat and calling|
```

**Comment by scanum** [comment]
@bjwhalen and @dearbeen - Any updates on this? Thanks.

**Comment by scanum** [comment]
@LaszloSomi Hi Laszlo, could you please review this PR and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@LaszloSomi - please review. Your earlier PR may have already addressed this.

**Comment by LolaJacobsen** [comment]
@bjwhalen - could you review this PR please?

**Comment by scanum** [comment]
@LolaJacobsen @bjwhalen Hi, any updates on this? Thanks.

**Comment by LolaJacobsen** [comment]
I ping'd @bjwhalen in chat.

**Comment by bjwhalen** [comment]
This page is way more complicated than it needs to be.  Given this page I can much better appreciate why people thing modes are complicated.  I believe this page needs a total re-write, as it need not be so complicated. we really just need some basic pros and cons. 

As for the specific change being discussed here, both the original content and the proposed content miss the fundamental point that TeamsOnly mode should be the destination for every single org in the long run. Some orgs will get there sooner than others, and some users in a given org will get there before other users in the same org.   This is true whether the org is pure online or hybrid.   The experience of using Teams when in Islands is not the same and not nearly as good when in TeamsOnly mode.  Being TeamsOnly ensures that messages and chats from any users, whether that users is in sfb onprem, sfb online, teams, whether same org or different org or whether pstn call -always lands in Teams for that user.  That simply cannot be said for islands and it's not even close.  

we must not set expectations that islands mode is a long term steady state.  every customer must be making a plan to get off islands mode.

File: Teams/upgrade-and-coexistence-of-skypeforbusiness-and-teams.md (hunk 1 above)
Comment (bjwhalen) [reply Teams/upgrade-and-coexistence-of-skypeforbusiness-and-teams.md]: See my comments in the discussion.

**Comment by LolaJacobsen** [comment]
We're going to rewrite this article. I made several small changes in a separate PR, but I'm not merging this one.

**Pull request closed without merging**